sktime/annotation/tests/test_plotting.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from sktime.annotation.plotting.utils import (
+    plot_time_series_with_change_points,
+    plot_time_series_with_profiles,
+)
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+
+@pytest.fixture
+def time_series_data():
+    ts_data = np.random.rand(100)
+    ts = pd.DataFrame({"Data": ts_data})
+    true_cps = [4, 8]
+    font_size = 12
+    ts_name = "Test Time Series"
+    profiles = np.array([np.random.rand(100) for _ in range(20)])
+    found_cps = [35, 65]
+    score_name = "Custom Score"
+    return {
+        "ts_name": ts_name,
+        "ts": ts,
+        "true_cps": true_cps,
+        "font_size": font_size,
+        "profiles": profiles,
+        "found_cps": found_cps,
+        "score_name": score_name,
+    }
+
+
+@pytest.mark.skipif(
+    not _check_soft_dependencies("matplotlib", severity="none"),
+    reason="Seaborn is required as a dependency for this plot.",
```

Comment:
Doubt: why does the message mention `seaborn` and the check is done for `matplotlib`? Shouldn't you check for `seaborn` instead?

Reply:
I fixed it